docs/whatsnew/6.0.rst

```diff
@@ -172,8 +128,38 @@ New `~sunpy.coordinates.PlanarScreen` context manager
 The plane goes through Sun center (or some specified distance from Sun center) and is perpendicular to the vector between the specified vantage point and Sun center.
 This replaces the default assumption where 2D coordinates are mapped onto the surface of the Sun and is an alternative to `~sunpy.coordinates.SphericalScreen`.
 
+Fix filename sanitization for downloaded files from the VSO
+===========================================================
+
+The `.VSOClient` has been sanitizing filenames to ensure that they are valid on all platforms.
+However, we have now fixed the sanitization to be more conservative, to follow the NFKC Unicode normalization, which is the recommended normalization for filenames on most platforms.
+This now does not replace periods, does not change letter case and does not leave Unicode characters decomposed.
+One example is that the letter "ä" will now be preserved in the filename.
+Another example is that the filename "aia.lev1.171A_2020_06_07T06_33_09.35Z.image_lev1.fits" will be preserved instead of being replaced with "aia_lev1_171a_2020_06_07t06_33_09_35z_image_lev1.fits".
+
+Arguments for :meth:`~sunpy.map.GenericMap.reproject_to`
```

Comment:
I would cut this from the what's new, as it's not a new feature or major bug fix?

Reply:
Removed